mapper.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 var (
-	identifierRE = `[a-zA-Z_][a-zA-Z0-9_]+`
-	metricLineRE = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
-	labelLineRE  = regexp.MustCompile(`^(` + identifierRE + `)\s*=\s*"(.*)"$`)
-	metricNameRE = regexp.MustCompile(`^` + identifierRE + `$`)
+	identifierRE      = `[a-zA-Z_][a-zA-Z0-9_]*`
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)+(\*|[^*.]+)$`)
```

Comment:
Actually (and this error was there before), this regex forbids single-component metrics. Should it be:

    ^(\*\.|[^*.]+\.)*(\*|[^*.]+)$

(`*` in the middle instead of `+`)